### ISSUE TITLE
Fields called `prev` and `next` are removed from response objects in 1.2.13+

### ISSUE THREAD

**Issue by lojewalo** [HoudiniGraphql/houdini#1192]
### Describe the bug

I have a query that looks like this

```graphql
query GetFeaturedMods($page: Int!, $count: Int!, $filterInfo: FilterInfo) {
    featuredPackages(page: $page, count: $count, filterInfo: $filterInfo) {
        pageInfo {
            prev
            next
        }

        packages {
            // ...
        }
    }
}
```

The actual JSON response looks like this

```jsonc
{
  "data": {
    "featuredPackages": {
      "pageInfo": {
        "prev": false,
        "next": true
      },
      "packages": [
        // ...
      ]
    }
  }
}
```

However, starting in Houdini 1.2.13, I get this as the response from Houdini.

```
Object { featuredPackages: {…} }
    featuredPackages: Object { pageInfo: {}, packages: (1) […] }
    <prototype>: Object { … }
```

Note that the `prev` and `next` keys are fully missing. In 1.2.12 and all previous versions, the object is populated as expected. I understand Houdini has its own pagination system, so maybe that's causing some interference, but I can't imagine that's intentional!

### Severity

blocking an upgrade

### Steps to Reproduce the Bug

1. Run a query as listed above from a `+page.ts`
```ts
const featured = load_GetFeaturedMods({
    variables: {
        page: state.featuredPage || 0,
        count: 6,
    },
    event,
});

const data = await loadAll(
    // other queries go here as well
    featured,
);

return {
    ...data,
    // other stuff
};
```
2. Try to access the `pageInfo` object

**Comment by lojewalo** [comment]
https://github.com/HoudiniGraphql/houdini/commit/41e3bdbf9a1bcc029fb8ef56fe91f7116a42a3b1#diff-8983c33c5d66912d4d0a408268e26c6d0b7242f4fd40a291ff5ee4121689dbb7

This looks like the likely cause.

**Comment by AlecAivazis** [comment]
🤦  yep! Good catch - i'll get something out now

**Comment by AlecAivazis** [comment]
Fix will be deployed in `v1.2.22`  once #1194  is merged
